spec/presenters/candidate_interface/application_form_presenter_spec.rb

```diff
@@ -55,6 +55,22 @@
     end
   end
 
+  describe '#personal_details_valid?' do
```

Comment:
Can we break these specs into two contexts?

Reply:
@stevehook yes, exactly. It makes it clearer and quicker to understand. The presenter can also be a subject in the spec (but I don't mind as it is above too). :)